schemas/hygiene/aep-hygiene-ops-record.example.1.json

```diff
@@ -0,0 +1,27 @@
+{
+  "xdm:batchTime": "2022-03-03T18:57:00+00:00",
+  "xdm:operationType": "identityDelete",
```

Comment:
Please make this `"xdm:action": "delete_identity"`.

Reply:
Done